python/rikai/pytorch/models/fasterrcnn_resnet50_fpn.py

```diff
@@ -12,12 +12,28 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import torchvision
+
 from .torchvision import ObjectDetectionModelType
+from rikai.mixin import Pretrained
+from rikai.spark.sql.model import ModelSpec
+from rikai.spark.sql.codegen.dummy import DummyModelSpec
 
 __all__ = ["MODEL_TYPE"]
 
 
-class FasterRCNNModelType(ObjectDetectionModelType):
+class FasterRCNNModelType(ObjectDetectionModelType, Pretrained):
```

Comment:
In this class, `load_pretrained_model` and `load_model` both have a name in pattern `load_xx`, but the first one just returns a model to the caller, the second one will change the property of `self`. Can we design a name pattern that can help users to distinguish the kind of activities from the pattern of method names?

I propose we can name the first kind of activity `fetch_xx` that just returns a value and will not change `self`, `load_xx` are methods that will change the content of `self`.

Reply:
see the latest commit